elliottlib/cli/find_builds_cli.py

```diff
@@ -366,38 +370,43 @@ def _fetch_builds_by_kind_rpm(runtime: Runtime, tag_pv_map: Dict[str, str], brew
 
     green_prefix('Generating list of rpms: ')
     click.echo('Hold on a moment, fetching Brew builds')
-
-    builder = BuildFinder(brew_session, logger=LOGGER)
     builds: List[Dict] = []
-    for tag in tag_pv_map:
-        # keys are rpm component names, values are nvres
-        component_builds: Dict[str, Dict] = builder.from_tag("rpm", tag, inherit=False, assembly=assembly, event=runtime.brew_event)
-
-        if runtime.assembly_basis_event:
-            # If an assembly has a basis event, rpms pinned by "is" and group dependencies should take precedence over every build from the tag
-            el_version = isolate_el_version_in_brew_tag(tag)
-            if not el_version:
-                continue  # Only honor pinned rpms if this tag is relevant to a RHEL version
-
-            # Honors pinned NVRs by "is"
-            pinned_by_is = builder.from_pinned_by_is(el_version, runtime.assembly, runtime.get_releases_config(), runtime.rpm_map)
-            _ensure_accepted_tags(pinned_by_is.values(), brew_session, tag_pv_map)
-
-            # Builds pinned by "is" should take precedence over every build from tag
-            for component, pinned_build in pinned_by_is.items():
-                if component in component_builds and pinned_build["id"] != component_builds[component]["id"]:
-                    LOGGER.warning("Swapping stream nvr %s for pinned nvr %s...", component_builds[component]["nvr"], pinned_build["nvr"])
-
-            component_builds.update(pinned_by_is)  # pinned rpms take precedence over those from tags
-
-            # Honors group dependencies
-            group_deps = builder.from_group_deps(el_version, runtime.group_config, runtime.rpm_map)  # the return value doesn't include any ART managed rpms
-            # Group dependencies should take precedence over anything previously determined except those pinned by "is".
-            for component, dep_build in group_deps.items():
-                if component in component_builds and dep_build["id"] != component_builds[component]["id"]:
-                    LOGGER.warning("Swapping stream nvr %s for group dependency nvr %s...", component_builds[component]["nvr"], dep_build["nvr"])
-            component_builds.update(group_deps)
-        builds.extend(component_builds.values())
+
+    if member_only:  # Sweep only member rpms
+        for tag in tag_pv_map:
+            builds_for_tag = asyncio.get_event_loop().run_until_complete(asyncio.gather(*[exectools.to_thread(progress_func, functools.partial(rpm.get_latest_build, default=None, el_target=tag)) for rpm in runtime.rpm_metas()]))
+            builds.extend(filter(lambda b: b is not None, builds_for_tag))
```

Comment:
`builds.extend(list(filter(None, builds_for_tag)))`

Reply:
Personally I prefer to explicitly check if the return value is None here.
Also there is no need to convert the return value to list.